hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/JdbcSinkBuilder.java

```diff
@@ -33,10 +35,21 @@
  * @since 4.1
  */
 public class JdbcSinkBuilder<T> {
+    /**
+     * The default batch size limit to use for the sink if batching is supported.
+     */
+    public static final int DEFAULT_BATCH_LIMIT = 50;
+
+    /**
+     * The default setting for whether exactly-once is allowed for the sink.
+     */
+    public static final boolean DEFAULT_EXACTLY_ONCE = true;
```

Comment:
We don't need to add this to the public API, it's too trivial. I mean, add `DEFAULT_BATCH_LIMIT`, but don't add `DEFAULT_EXACTLY_ONCE`.

Reply:
I added that just for general consistency and for the code to not look ugly having one constant and one literal here:
```java
    private int batchLimit = DEFAULT_BATCH_LIMIT;
    private boolean exactlyOnce = DEFAULT_EXACTLY_ONCE;
```
Also, we can reference the default in the javadoc via the `@value` annotation cleanly too.

Let me know if you still prefer to leave it as it was. No worries.